.github/actions/test/action.yaml

```diff
@@ -24,3 +27,8 @@ runs:
     - name: Test package
       run: uds run test-package
```

Comment:
question: Should this 'Test package' and the next 'Test Package' step be an either/or? Meaning this one should be "if not inputs.upgrade"?

Reply:
I see how it works in the gitlab PR now. Resolving this!